components/layout/Layout.tsx

```diff
@@ -159,8 +158,7 @@ const Layout = ({ children, toggleTheme, isDarkTheme }: LayoutProps) => {
       </Wrapper>
 
       <Main>{children}</Main>
-      <Footer />
```

Comment:
So here, we need the `Footer` component to stay so the app will render it on every page. The `Main` tag above is where the main body of the app gets dynamically put in/changed.